homeassistant/components/hassio/backup.py

```diff
@@ -212,10 +215,6 @@ async def async_delete_backup(self, backup_id: str, **kwargs: Any) -> None:
                     location={self.location}
                 ),
             )
-        except SupervisorBadRequestError as err:
-            if err.args[0] != "Backup does not exist":
-                raise
-            _LOGGER.debug("Backup %s does not exist", backup_id)
```

Comment:
This no longer happens, `aiohasupervisor` converts this to `SupervisorNotFoundError`